packages/theme/composables/useCart/useCart.ts

```diff
@@ -79,7 +79,7 @@ export interface UseCartInterface<CART, CART_ITEM, PRODUCT> {
    *
    * This is `true` when loading, saving, updating, or removing customer addresses and `false` otherwise.
    */
-  loading: DeepReadonly<Ref<boolean>>;
+  loading: Readonly<Ref<boolean>>;
```

Comment:
With this change:
![image](https://user-images.githubusercontent.com/5359825/165515795-e2148181-f639-4a70-bf8c-73da6f8942f4.png)


Without this change (previous):
![image](https://user-images.githubusercontent.com/5359825/165515678-d2237ffa-808a-432d-9a6a-349ac544af5d.png)

Should stay as is imo
```suggestion
  loading: DeepReadonly<Ref<boolean>>;
```

Reply:
![Screenshot 2022-04-27 at 14 52 25](https://user-images.githubusercontent.com/16045377/165522239-5d895cfd-4182-461d-8ca9-84983f47c7db.png)

And Readonly perfectly align with the loading data structure.

/**
 * Make all properties in T readonly
 */
type Readonly<T> = {
    readonly [P in keyof T]: T[P];
};

Reply:
IDK why do you have `const loading: any` in the first place